azure-pipelines.yml

```diff
@@ -397,7 +397,33 @@ jobs:
           # Note: this is going to get the PR branch commit, not the
           # result of the merge (i.e. this is not using the same commit as the
           # other jobs in this build).
-          git fetch origin $(branch_sha)
+          #
+          # We have seen errors getting this commit from GitHub, which we
+          # suspect are transient network errors, so adding simple exponential
+          # backoff to mitigate.
+          BACKOFF=1
+          tell_gary() {
```

Comment:
I would strongly recommend to not add that. You have more than enough things on your plate already :slightly_smiling_face: But I’ll leave it to up to you.

Reply:
I'm not planning to jump on each of those, but I do want to get a sense for how much of an issue this is, as not everyone reports it when it happens.